.eslintrc.js

```diff
@@ -55,6 +55,7 @@ module.exports = {
         }
       }
     ],
+    'newline-per-chained-call': ['error', { ignoreChainWithDepth: 2 }],
```

Comment:
It seems like you may also need something like this to enforce proper indenting, but you'd need to test it.
```
'indent': ['error', 2],
```

Reply:
Yes, the indentation seemed odd to me but adding this `'indent': ['error', 2],` will affect the entire codebase which means some of the existing codes will be changed too. If you're okay with this I will commit the changes.
![Screenshot 2024-10-09 192115](https://github.com/user-attachments/assets/0c8aa154-7335-4f9d-a197-e1dc758536f8)

Reply:
That feels ok with me, but test it and we’ll let @itsalaidbacklife have final say.

Reply:
> That feels ok with me, but test it and we’ll let @itsalaidbacklife have final say. Seems like a bit of research is needed first.

Is this okay? @seriouslysean @itsalaidbacklife

Reply:
Yes I think that's what we want. We'll have to see how it goes and it's plausible that we may need to tweak or backpedal at that point but let's try it and see

Reply:
I've already committed the changes... You may review them

Reply:
Let's additionally configure the SwitchCase option to preserve the single level of indentation applied to switch cases. It'll mean using the object definition of the rule and setting 

`SwitchCase: 1`